src/customers/customer-account-manage.md

```diff
@@ -99,6 +99,20 @@ See [Managing Subscribers]({% link marketing/newsletter-subscribers.md %}) for m
 
    To learn more, see [Updating Customer Accounts]({% link customers/customer-account-update.md %}).
 
+## Export customers accounts
+
+1. On the _Admin_ sidebar, go to **Customers** > **All Customers**.
+
+1. In the action menu, click Export and select the desired format:
+
+- CSV
+
+- Excel XML
+
+1. Click <span class="btn">OK</span>.
+
+1. Open the exported file in the browser downloads section.
+
```

Comment:
There should be a note that this will export EVERY customer. If they want to export a limited number of accounts, they should either tick the ones they want to export OR use the filters to select a range of customer accounts and then use the export.